components/LatestNews.tsx

```diff
@@ -0,0 +1,46 @@
+import { titleFont } from "@/pages/_app";
+import styles from "@/styles/LatestNews.module.css";
+import Card from "./general/Card/Card";
+
+const LatestNews = () => {
+  const newsList = [
+    {
+      id: 1,
+      image: "/images/two_people_praying.png",
+      cornerMakerLeft: "Branding, Design",
+      title: "Things to Look for When Comparing Branding Alternatives",
+    },
+    {
+      id: 2,
+      image: "/images/channel.png",
+      cornerMakerLeft: "Branding, Design",
+      title: "5 Stand-out Features of Branding You Should Know",
+    },
+    {
+      id: 3,
+      image: "/images/brand_strategy.png",
+      cornerMakerLeft: "Branding, Design",
+      title: "Branding: What Real Customers Have to Say",
+    },
+  ];
+
+  return (
+    <section className={styles.last_news}>
+      <header className={styles.last_news__header}>
```

Comment:
If you are planning to use a header tag, where is the article tag outside?

Reply:
The outside article tag is unnecessary here, so I changed it back to "div" tag.